designs/simplified-scaffolding.md

```diff
@@ -417,7 +417,9 @@ func init() {
 }
 
 func main() {
-	ctrl.SetLogger(zap.Logger(true))
+	ctrl.SetLogger(zap.New(func(o *zap.Options) {
```

Comment:
This is really pretty hacky -- let's avoid making users do this in the future -- we should have a pre-made option for development mode, and we shouldn't be scaffolding things out with something hacky like this.  Remember, scaffolding doesn't get to change, so if we scaffold out something clunky, there'll be a whole set of users that get this code.

Reply:
Filed PR for CR: https://github.com/kubernetes-sigs/controller-runtime/pull/646